manifests/params.pp

```diff
@@ -21,20 +21,25 @@
   $use_scl_ruby = true
   $scl_ruby = 'rh-ruby25'
 
-  $scl_nodejs = 'rh-nodejs12'
-  $scl_postgresql = 'rh-postgresql12'
+  if $facts['os']['release']['major'] == '7' {
+    $scl_nodejs = 'rh-nodejs12-'
+    $scl_postgresql = 'rh-postgresql12-'
+  } else {
+    $scl_nodejs = undef
+    $scl_postgresql = undef
+  }
 
   $manage_bundler = true
 
   $initial_organization = 'Default Organization'
   $initial_location = 'Default Location'
   $admin_password = 'changeme'
-  $enable_ostree = true
+  $enable_ostree = $facts['os']['release']['major'] == '7'
```

Comment:
Perhaps it's better to include `katello::params` and reuse `$katello::params::pulp2_support` for this?

Reply:
That threw all kinds of duplicate declaration errors so I just opted to stay with the easy path.